server/sonar-server/src/main/java/org/sonar/server/issue/IssueService.java

```diff
@@ -279,24 +280,6 @@ public DefaultIssue createManualIssue(String componentKey, RuleKey ruleKey, @Nul
     }
   }
 
-  // TODO result should be replaced by an aggregation object in IssueIndex
```

Comment:
In fact I use the travis build to track all test failures related to timezone coupling in tests. I should have done a second PR.